src/test/java/com/bloomtech/asylumrgbea/controllers/AsylumCaseControllerTest.java

```diff
@@ -42,23 +42,47 @@ void getAllCases_serviceIsValid_returnsIterableOfAsylumCaseResponseDto() {
         verify(asylumCaseService).getAllAsylumCases();
     }
 
-    @Test
+    @Deprecated
     void getPageOfAsylumResponse_serviceIsValid_returnsIterableOfPageResponseDto() {
-//        // GIVEN
-//        AsylumCaseRequestDto asylumCaseRequestDto = new AsylumCaseRequestDto(10, 0);
-//        AsylumCaseResponseDto asylumCaseResponseDto = new AsylumCaseResponseDto(
-//                "SAN", "HIO", "Other",
-//                "Pending", "N/A", "6/24/2022");
-//
-//        PageResponseDto pageResponseDto = new PageResponseDto(2, List.of(asylumCaseResponseDto));
-//
-//        // WHEN
-//        when(asylumCaseService.getPageOfAsylumCases(new AsylumCaseRequestDto())).thenReturn(pageResponseDto);
-//        PageResponseDto result = asylumCaseController.getPageOfCases(asylumCaseRequestDto);
-//
-//        // THEN
-//        assertNotNull(pageResponseDto.getPage());
-//        assertNotNull(pageResponseDto.getPage());
-//        verify(asylumCaseService).getPageOfAsylumCases(any());
+        // GIVEN
+        AsylumCaseRequestDto asylumCaseRequestDto = new AsylumCaseRequestDto(10, 0);
+        AsylumCaseResponseDto asylumCaseResponseDto = new AsylumCaseResponseDto(
+                "SAN", "HIO", "Other",
+                "Pending", "N/A", "6/24/2022");
+
+        PageResponseDto pageResponseDto = new PageResponseDto(2, List.of(asylumCaseResponseDto));
+
+        // WHEN
+        when(asylumCaseService.getPageOfAsylumCases(new AsylumCaseRequestDto())).thenReturn(pageResponseDto);
+        PageResponseDto result = asylumCaseController.getPageOfCases(asylumCaseRequestDto);
+
+        // THEN
+        assertNotNull(pageResponseDto.getPage());
+        assertNotNull(pageResponseDto.getPage());
+        verify(asylumCaseService).getPageOfAsylumCases(any());
+    }
+
+    void getCases_withNoQueryParameters_returnsAllCases() {
+        // TODO: AsylumCaseServiceImpl has not been scoped-redesigned.
+    }
+
+    void getCases_withSingleQueryParameter_returnsCasesMatchingSingleQueryParameter() {
+        // TODO: AsylumCaseServiceImpl has not been scoped-redesigned.
+    }
+
+    void getCases_withMultipleQueryParameter_returnsCasesMatchingMultipleQueryParameters() {
+        // TODO: AsylumCaseServiceImpl has not been scoped-redesigned.
+    }
+
+    void getCases_withSingleQueryParameterNoMatches_returnsErrorDto() {
```

Comment:
This division of potential outcomes will prove helpful in fleshing out frontend handling of ErrorDTOs. A consideration for implementation, when that happens, should be whether the query parameters provided are viable values -- we have discreet values for most of the potential inputs, and so validating these values on the backend may prove invaluable in detecting aberrations between front end and back end schema.